files/default/build_cookbook/kitchen.yml

```diff
@@ -11,8 +11,8 @@ provisioner:
   product_name: chefdk
```

Comment:
Should the default `kitchen.yml` have the multiple_converge / idempotency checks enabled?
```yaml

provisioner:
  name: chef_solo
  enforce_idempotency: true
  multiple_converge: 2
  deprecations_as_errors: true
```